scripts/timer.js

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+
+var exec = require('child_process').exec;
+var fs = require('fs');
+
+var name = process.argv[2];
+var cmd = process.argv.slice(3).join(' ');
+var start = +new Date();
```

Comment:
I would prefer `Date.now()` for readability instead of depending on type conversion using `+`.  Don't make me think so hard :-P